pom.xml

```diff
@@ -247,6 +247,12 @@
 				<version>${caffeineVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>fhir2-omod</artifactId>
+				<version>1.0.0-SNAPSHOT</version>
```

Comment:
We should update this module to use 1.2.1 or 1.3.0-SNAPSHOT.

Reply:
I will change this. I think version 1.2.1 will be better because it is more stable. Thought?